src/main/plugin/dcat-ap/index-fields/index.xsl

```diff
@@ -310,16 +269,20 @@
   </xsl:template>
 
   <xsl:template mode="index-keyword" match="dcat:Dataset|dcat:DataService">
```

Comment:
Can we remove this and rely on https://github.com/metadata101/dcat-ap/blob/feature/documentation-how-to-profile/src/main/plugin/dcat-ap/index-fields/index.xsl#L340-L341 adding the tag field in there.

I'm wondering if we can even remove the tag field which is the aggregations of all keywords by using a copy_to directive in https://github.com/geonetwork/core-geonetwork/blob/main/web/src/main/webResources/WEB-INF/data/config/index/records.json#L1361-L1364 - I'm testing

Reply:
copy_to do not handle that https://www.elastic.co/guide/en/elasticsearch/reference/current/copy-to.html and tagNumber would be hard to accumulate values

Reply:
Maybe not super important because record view rely on allThesaurus and when using facets, th_* are used. So tag is mainly used as a generic facet which is quite often replaced by th_* facets in the end.